Added 5 examples to demostrate GPU aware scheduling features (#150)
intel/platform-aware-scheduling: gpu-aware-scheduling/docs/example/README.md

> This folder have simple examples which uses kubernetes extended resources and it features.

Reviewer: ```suggestion
This folder has simple examples which use kubernetes extended resources and other GAS features.
```
